### PR TITLE
#2188: Register quickbar with `chrome.commands`

### PR DESCRIPTION
**Pull request by fregante**
- Fixes https://github.com/pixiebrix/pixiebrix-extension/issues/2188

This setup appears to work, with the exception that KBar is not actually created/registered until its regular shortcut is triggered. I'd need some React help here.

## Setup

1. Open `chrome://extensions/`
2. Open burger menu on the top left
3. Open "Keyboard shortcuts"
4. Set you new shortcut, different from cmd-k. I used cmd-shift-l

## Hosts with Quick Bar already registered

This sequence works:

1. Press cmd-k
2. Press your global shortcut

## Hosts without permissions

This sequence works:

1. Press your global shortcut (activates tab + injects content script)
1. Press cmd-k
2. Press your global shortcut

In short, I need help to:

- [x] Create the kbar component even without triggering the native cmd-k command

### CODE REVIEW

**Review by twschiller — APPROVED** [review]
@fregante should be working now. I think there's a race condition depending on which method calls the init

**Comment by fregante** [comment]
I'll test it out

**Comment by fregante** [comment]
This seems to work correctly with either shortcut. The race condition I see is that the stylesheet no longer has enough time to load before kbar is shown, so the issue described in https://github.com/pixiebrix/pixiebrix-extension/pull/2272#issuecomment-1003049856 appears on the first global-shortcut activation:

Screen Shot 4

Solution: A shadowdom-compatible React component that delays the display of `KBarComponent` until the stylesheet has loaded.
